### DISCUSSION TITLE
Auth Pricing for Pay as You go

### DISCUSSION THREAD

**Discussion by derelict463** [supabase/supabase#1957] [Questions] [answered]
Reading through the pricing docs, everything else seems pretty straightforward - where price is incremented based on usage, but for auth it simply says "unlimited". What would the pricing look like if you use all the other resources within pro tier but Auth gets higher - around lets say 500k? Is there a xx per 100,000 users payment we should think about? Thanks

**Comment by inian** [comment, marked as answer]
Hey @derelict463, we do not plan on charging for Auth users in the unlimited plan. The user data is completely stored on your own database and in the higher plans, the cost of Auth users will be blended in with other metrics we price on like Database space, etc.
